client/src/config.rs

```diff
@@ -20,7 +20,7 @@ use crate::{
 mod user;
 
 #[allow(missing_docs)]
-pub const DEFAULT_TRANSACTION_TIME_TO_LIVE: Duration = Duration::from_secs(100);
+pub const DEFAULT_TRANSACTION_TIME_TO_LIVE: Duration = Duration::from_secs(1000);
 #[allow(missing_docs)]
 pub const DEFAULT_TRANSACTION_STATUS_TIMEOUT: Duration = Duration::from_secs(15);
```

Comment:
We set default ttl to 1000 seconds, but client will wait only for 15 seconds for response, should this value be increased as well.

Also do you thinks ~15 minutes is reasonable default value for general case?

Reply:
I undid the change and I think it may have even the torii api tests.